kolibri/plugins/audio_mp3_render/assets/src/render_audio_mp3_module.js

```diff
@@ -0,0 +1,23 @@
+'use strict';
+
+var _ = require('lodash');
+var logging = require('loglevel');
+
+// This is aliased for your convenience!
+var KolibriModule = require('kolibri_module');
+
+var AudioMP4Module = KolibriModule.extend({
+
+    render: function(content_data, render_area) {
```

Comment:
I think most of these plugins should be kept, but the actual render methods can be removed for proper fill in with appropriate Vue logic later.